Automattic/wp-calypso
# GMB: Show GMB nudge after site is 2 weeks old in 2 weeks intervals

We need to be able to display this nudge after a certain amount of time has passed (let's start with a week) since the site was created. We also need to display it again one more time two weeks after that event.
  
#### Testing instructions
  
1. Run `git checkout add/time-interval-gmb-nudge` and start your server, or open a [live branch](https://calypso.live/?branch=add/time-interval-gmb-nudge)
2. Create a site with a `promote` siteGoals if you don't have one. This can be done from the `http://calypso.localhost:3000/start/about` page by selecting at least the "Promote your business, skills, organization, or events" option.
3. Navigate to this site stats page `/stats/day/:site_slug`
4. Notice the default stat page
5. Upgrade your site plan to business or premium, and change creation date to be in the past ( more than 2 weeks )
6. Go back to the stats page
7. Notice the GMB nudge
  
#### Reviews
